docs/README.md

```diff
@@ -23,9 +23,13 @@ Each Campaign consists of one or more Collections. You can view those Collection
 <img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image3.jpg" width="auto">
 
 ## Selecting an Item
-Within the Collection, you will be presented with a description of that Collection and the Items it contains. You also have the option to view or hide the Items you have already annotated, by selecting the ***View*** or ***Hide*** option on the selector.
+Within the Collection, you will be presented with a description of that Collection and the Items it contains. You can choose which items you want to see, by selecting one of the *ALL ITEMS*, *MY ANNOTATED ITEMS*, *NOT ANNOTATED ITEMS* options on the selector.
 
-<img src="https://raw.githubusercontent.com/ails-lab/CrowdHeritage_Documentation/main/docs/_media/image4.jpg" width="auto">
+<img src="_media/collection-editor-view-options.png" width="auto">
```

Comment:
Newly recorded images I added them with a relative path so that we can preview them locally.

I expect this to work for deployment as well, as with normal websites but will double check upon deployment. 

Had trouble installing Docsify locally and did not want to spend time.